fillpdf.go

```diff
@@ -172,12 +194,13 @@ func createFdfFile(form Form, path string) error {
 	// Write the form data.
 	var valueStr string
 	for key, value := range form {
-		// Convert to Latin-1.
-		valueStr, err = latin1Encoder.String(fmt.Sprintf("%v", value))
+		// First converts the FormOption value to the correct fdf string representation.
+		// Then converts string to Latin-1.
+		valueStr, err = latin1Encoder.String(fmt.Sprintf("%v", value.String()))
```

Comment:
I think this can be simplified

```suggestion
		valueStr, err = latin1Encoder.String(value.String())
```